ironfish-cli/src/commands/wallet/which.ts

```diff
@@ -6,11 +6,11 @@ import { IronfishCommand } from '../../command'
 import { RemoteFlags } from '../../flags'
 
 export class WhichCommand extends IronfishCommand {
-  static description = `Show the account currently used.
+  static description = `Show the wallet currently used.
 
-  By default all commands will use this account when deciding what
-  keys to use. If no account is specified as the default, you must
-  specify the account in the command using --account <name>`
+  By default all commands will use this wallet when deciding what
+  keys to use. If no wallet is specified as the default, you must
+  specify the wallet in the command using --account <name>`
```

Comment:
This is incorrect: this command is used to determine which account is in use. The wallet can contain multiple accounts.